activerecord/test/cases/timestamp_test.rb

```diff
@@ -425,6 +425,17 @@ def test_all_timestamp_attributes_in_model
     toy = Toy.first
     assert_equal [:created_at, :updated_at], toy.send(:all_timestamp_attributes_in_model)
   end
+
+  def test_index_is_created_for_both_timestamps
+    connection = ActiveRecord::Base.connection
+
+    if connection.respond_to?(:indexes)
+      indexes = connection.indexes("flowers")
+      assert_equal ["created_at", "updated_at"], indexes.flat_map(&:columns).sort
+    else
+      warn "#{connection.class} does not respond to #indexes"
```

Comment:
I took this from https://github.com/rails/rails/blob/master/activerecord/test/cases/adapter_test.rb#L42 but agree that it's a little too defensive: it has now been removed.